generate_rss.py

```diff
@@ -31,6 +36,63 @@ def findall(self, *args, **kwargs):
 entry.formatRFC2822 = feed.formatRFC2822 = _format_rfc_2822
 line_cache: dict[Path, dict[str, str]] = {}
 
+# Monkeypatch PEP and RFC reference roles to match Sphinx behaviour
+EXPLICIT_TITLE_RE = re.compile(r'^(.+?)\s*(?<!\x00)<(.*?)>$', re.DOTALL)
+def _pep_reference_role(role, rawtext, text, lineno, inliner,
```

Comment:
Do we need to copy/paste this here (at least, I'm assuming that's what we're doing), or can we import it from its current location or a common `utils` module?

Reply:
From my response above (didn't see this question):

> All the code was copied from the same backport I used in `pep2html`. 

A

Reply:
Hmm, I guess that makes sense.